### ISSUE TITLE
set failed: The client is not authenticated with the Multipass service. (windows11)

### ISSUE THREAD

**Issue by Dexus** [canonical/multipass#3365]
**Describe the bug**
```
C:\Windows\System32>multipass authenticate
Please enter passphrase:
authenticate failed: Passphrase is not set. Please `multipass set local.passphrase` with a trusted client.

C:\Windows\System32>multipass set local.passphrase
Please enter passphrase:
Please re-enter passphrase:
set failed: The client is not authenticated with the Multipass service.
Please use 'multipass authenticate' before proceeding.

C:\Windows\System32>multipass authenticate
Please enter passphrase:
authenticate failed: Passphrase is not set. Please `multipass set local.passphrase` with a trusted client.
```
**To Reproduce**
How, and what happened?
1. install multipass
2. try to start multipass client not allowed to connect

**Expected behavior**
working multipass

**Logs**
Please provide logs from the daemon, see [accessing logs](https://multipass.run/docs/accessing-logs) on where to find them on your platform.

**Additional info**
 - OS: [e.g. macOS 10.15] Windows 11 latest
- `multipass version` multipass   1.12.2+win
- `multipass info --all` 
```
multipass info --all
info failed: The client is not authenticated with the Multipass service.
Please use 'multipass authenticate' before proceeding.
```
- `multipass get local.driver`
```
C:\Users\xxx>multipass get local.driver
get failed: The client is not authenticated with the Multipass service.
Please use 'multipass authenticate' before proceeding.
```
**Additional context**
Nothing works no user no admin...

I can see the service fail with `powershell.exe takeown /a /r /d Y /f "C:\ProgramData\Multipass"` because its return exit code 1
and some "network error" but network works fine... even in browser it opens:
```
[url downloader] Error getting https://cdimage.ubuntu.com/ubuntu-core/appliances/streams/v1/com.ubuntu.appliances:released:download.json: Temporary network failure. - trying cache.
```
and some other similar errors to download json files...

**Comment by ricab** [comment]
Hi @Dexus, only the first user/client to connect to the daemon gets automatically authenticated. After that, other users need a passphrase, which the first user needs to set. Did you install or run as an Admin user first, perhaps? Does `multipass list` work on an admin shell? If so, you'd have to `multipass set local.passphrase` in that admin shell.

**Comment by buelev** [comment]
I'm having this issue as well，But it didn't work ou

**Comment by buelev** [comment]
At the same time, the document could not be found, and the verification went into an endless loop, switching back and forth between "C:\Windows\System32>multipass authenticate
Please enter your password:
authenticate failed: No password is set. Please use a trusted client for "multipass set local.passphrase".

C:\Windows\System32>multipass to local.passphrase
Please enter your password:
Please re-enter your password:
set failed: The client is not authenticated with the Multipass service.
Before proceeding, use 'multipass authenticate'.
I've even reinstalled the win system, but it still doesn't work, it crashes

**Comment by Dexus** [comment]
It did not work, not as admin not as my main user.
I had to deinstall everything and to delete the folders. This was not allowed to my main user and higher rights... (maybe because of mainboard switch?) so I had to remove all folders of multipass and the VM storage. After an other restart the system was ready to reinstall multipass and works now.

**Comment by buelev** [comment]
It seems like you are encountering an issue with modifying the default storage location for Multipass on your Windows computer. Despite following the steps outlined in the provided documentation (https://multipass.run/docs/configure-multipass-storage#heading--windows), you are stuck in a verification loop when trying to operate with either regular or administrator privileges. You also mentioned that you set a local passphrase during your initial Multipass run, but it doesn't seem to have resolved the problem after changing the default storage location.

Here's a translated summary of your situation in English. If you have any specific questions or need further assistance, feel free to ask!

**Comment by cicorias** [comment]
is there a "what else is needed" to address moving the storage location and the proper steps?

**Comment by SMUEric1127** [comment]
I agree with @cicorias, why is it so buggy... Just feel so hard to have it use a different drive on ubuntu.

**Comment by arturs706** [comment]
I am experiencing the same issues after reinstalling multipass on macOS. Not have an idea how to get this now up and running. Have reinstalled multipass with brew as a package. Probably easier is to reinstall whole macOS rather than get this to be fixed.

**Comment by apotenza92** [comment]
> I am experiencing the same issues after reinstalling multipass on macOS. Not have an idea how to get this now up and running. Have reinstalled multipass with brew as a package. Probably easier is to reinstall whole macOS rather than get this to be fixed.

Got around this on MacOS by using `sudo -i` in terminal to be root. Then manually removed the two multipass folders from `/var/root/Application Support/` with a `rm -rf`. Then did a brew --zap uninstall of multipass and reinstall with brew. Works now.

**Comment by robertlagrant** [comment]
Is it worth reconsidering how this works? "First client wins" and needing to set a local password to run things locally seems like a recipe for endless support issues. Is there another way to solve the problem(s) these decisions are trying to solve?

**Comment by kizzlah** [comment]
Still having issues a year later. I read it was a bug that shipped with version 1.16.1 so I installed 1.15.1, but now I see that even that version isn't working. I've removed ever file related to Multipass, reinstalled with Homebrew and GH package, but nothing helps.

**Comment by xqjcool** [comment]
same issue. Have you fixed it?

**Comment by kizzlah** [comment]
> same issue. Have you fixed it?

I gave up and had Warp Terminal look into the issue on my MBP. I've only had issues with Multipass a few other times since first discovering it. I can't remember specifics at this moment, however, I remember having to have the daemon "reset" once or twice and another time I wasn't being recognized as having root or admin privileges despite everything being visible correct. I think for the latter situation I went with a full wipe of the software and started completely fresh. This probably isn't the most hopefully, for which I do apologize. I work up in the middle do the night and randomly checked my email.
